pagopa/io-app
# fix: [IA-134,IA-114] Improve accessibility of link-like components

## Short description
Update _transaction reason_ and _anything missing?_ links to use the `Link` component, so that a11y properties are applied correctly

**Please note** that the commit for IA-114 is [already present](https://github.com/pagopa/io-app/commit/7139c4a86b3e4b020874407e68deba5636b56353) on `master`, therefore it results in an empty commit. Tracking it in this PR, where it was tested and which will push the task forward.

## List of changes proposed in this pull request
- Use `Link` component instead of Touchables
- The only size for a Link is 16px, therefore in the second picture below you will notice a change in the UI. Summoning @thisisjp to validate it from a design standpoint 🙇 

## How to test
Check via Accessibility Inspector:
1. Wallet -> Cashback -> Your transactions -> inspect the header below the title
2. Wallet -> Show transactions -> select the first one -> inspect the `Full Reason` link

![Screenshot 2021-08-31 at 13 53 45](https://user-images.githubusercontent.com/2094604/131505711-a51771f7-9788-4302-8aa0-f9d3c128238a.png)

![Screenshot 2021-08-31 at 14 42 52](https://user-images.githubusercontent.com/2094604/131505647-28fce755-603a-48bd-8cbd-153234abf8d0.png)
